components/clp-package-utils/clp_package_utils/scripts/native/search.py

```diff
@@ -4,23 +4,27 @@
 import asyncio
 import datetime
 import logging
+import msgpack
```

Comment:
We don't have automated linting set up yet, but third-party libraries (msgpack, pymongo, and zstandard) should be separated from (and come after) Python's standard libraries.